tests/scripts/all.sh

```diff
@@ -55,6 +55,16 @@
 # Notes for maintainers
 # ---------------------
 #
+# The bulk of the code is organized into functions that follow one of the
+# following naming conventions:
+#  * pre_XXX: things to do before running the tests, in order.
+#  * run_XXX: independent components. They can be run in any order.
```

Comment:
I don't think `run_XXX` is correct here. You probably mean `component_TYPE_XXX`.

Reply:
Fixed by amending "Move the code into functions. No behavior change."